biogeochem/EDCohortDynamicsMod.F90

```diff
@@ -128,10 +131,10 @@ module EDCohortDynamicsMod
 
   !-------------------------------------------------------------------------------------!
 
-  subroutine create_cohort(currentSite, patchptr, pft, nn, hite, dbh,   &
+    
+  subroutine create_cohort(currentSite, patchptr, pft, nn, hite, coage, dbh,   &
```

Comment:
Yes, good point. I was trying to avoid confusion with patch age which I think is just 'age' in the code. I could make it cohort_age?

Reply:
I think it might be more bother than it's worth to change, given your point about avoiding confusion with patch age (and people do, indeed, get very confused about these things).